### DISCUSSION TITLE
Can we have user level jwt security in verdaccio?

### DISCUSSION THREAD

**Discussion by gaya301** [verdaccio/verdaccio#2171] [Q&A]
Requirement: 
I need to disable token verification for Jenkins build user. JWT token should be validated for other users. 
Is there any way to support user level jwt security config?

> security:
    api:
      legacy: false
      jwt:
       sign:
         expiresIn: 2m
       verify:
         expiresIn: 2m

**Comment by juanpicado** [comment]
@gaya301 unfortunately is not possible at this point.
